templates/careers/engineering.html

```diff
@@ -46,7 +45,7 @@ <h1>Engineering</h1>
         <p class="p-muted-heading">
           <span style="text-transform: capitalize">Europe&nbsp;&#124;&nbsp;Middle East&nbsp;&#124;&nbsp;Africa&nbsp;&#124;&nbsp;Americas&nbsp;&#124;&nbsp;APAC</span>
         </p>
-        <a href="#">Apply now&nbsp;&rsaquo;</a>
+        <a href="#available-roles">Apply now&nbsp;&rsaquo;</a>
```

Comment:
It's not so noticeable on small screens, but on desktop when you click this link it switches to the relevant tab, but the page doesn't jump back up, so it looks like it's a blank page until you realise what's happened.

Reply:
@sowasred2012 it jumps to the top of that section...

Reply:
This is the behaviour I see on Chrome and Firefox:

![tab](https://user-images.githubusercontent.com/2376968/68402296-01209200-0173-11ea-9047-f4ef53b53bcb.gif)